src/random.c

```diff
@@ -24,7 +24,7 @@ void generate_random_byte(void)
 
     while (random_step--) {
         random_byte
-            = (random_byte << 1) | (random_byte >> 7) ^ (PEEK(0xD6DE) & 0x01);
+            = (random_byte << 1) | ((random_byte >> 7) ^ (PEEK(0xD6DE) & 0x01));
```

Comment:
These `random` variables have global scope, maybe they should be `static`?

Reply:
Perhaps in a separate PR, see above?

Reply:
@hth313 If there's nothing more for this PR, would you please click approve? (We now require at least one approved review before merging PR's)

Reply:
I have approved, but I do not think it helps as I do not have commit rights on this repository.

Reply:
Thanks! Let's ping @lydon42 for help.

Reply:
Random could be replaced soon by neoTRNG "real" random...